airflow/jobs.py

```diff
@@ -1790,6 +1790,56 @@ class BackfillJob(BaseJob):
         'polymorphic_identity': 'BackfillJob'
     }
 
+    class _DagRunTaskStatus(object):
+        """
+        Internal status of the backfill job. This class is intended to be instantiated
+        only within a BackfillJob instance and will track the execution of tasks,
+        e.g. started, skipped, succeeded, failed, etc. Information about the dag runs
+        related to the backfill job are also being tracked in this structure,
+        .e.g finished runs, etc. Any other status related information related to the
+        execution of dag runs / tasks can be included in this structure since it makes
+        it easier to pass it around.
+        """
+
+        # TODO(edgarRd): https://issues.apache.org/jira/browse/AIRFLOW-1444
+
+        def __init__(self,
+                     started=None,
+                     skipped=None,
+                     succeeded=None,
+                     failed=None,
+                     not_ready=None,
+                     deadlocked=None,
+                     finished_runs=0,
+                     total_runs=0,
+                     ):
+            """
+            :param started: Dict of tasks already started
```

Comment:
Nit: you can remove "Dict of" since it's redundantly given by the type below. Better just explain what the keys/values represent in english.

Same for the other params.